### DISCUSSION TITLE
Posibility to only use for serve or build mode

### DISCUSSION THREAD

**Discussion by evilchis94** [sapphi-red/vite-plugin-static-copy#77] [Q&A] [answered]
Hey @sapphi-red, thanks for this amazing vite-plugin.

I need to use the plugin only for serve mode. I've already tried the following with no luck 

`plugins: [
      {
        ...viteStaticCopy({
          targets: [
            {
              src: __dirname + '/../httpdocs/css',
              dest: 'css'
            }
          ]
        }),
        apply: 'serve'
      },
			vue(),
		],`

Do you know how I can achieve this? Thanks!

**Comment by sapphi-red** [comment, marked as answer]
This plugin returns two plugins, one for serve mode and one for build mode.
https://github.com/sapphi-red/vite-plugin-static-copy/blob/602627d606472d97b63a655deaa3e3c9b57063ee/src/index.ts#L15-L18
You can filter out the plugin for build mode one.

**Reply by evilchis94** [reply]
Thanks, this helped me!
